casper/src/test/scala/coop/rchain/casper/EstimatorHelperTest.scala

```diff
@@ -88,14 +88,14 @@ class EstimatorHelperTest
     }
   }
 
-  it should "conflict if their deploys contain same channel in deployLog" in withStorage {
+  ignore should "conflict if their deploys contain same channel in deployLog" in withStorage {
     implicit blockStore => implicit blockDagStorage =>
       testConflict[Task] { deploy =>
```

Comment:
```
first deploy should do a produce on channel A
second deploy should do a consume on channel A
(this conflicts)
```
This is a huge oversimplification as this isn't always going to conflict because the produce and consume might not match. Hence I would argue that this test should be removed given how fine grained the tests in `MultiParentCasperMergeSpec` are going to be

Reply:
> the produce and consume might not match
remember we're in control of the code used for doing the deploy. The `testConfclits` funciton ensures they do match, by not specifying a pattern on the for.

MergeSpec can't test inclusion of payment logs until they're actually used, and is far heavier than this is. If it's such a pain - leave those ignored, I'll fix it when I come to it.

Reply:
Nah, I'll unignore them